pipelines/install_pipeline.yml

```diff
@@ -38,18 +41,21 @@
       scenario_version: "{{ pipeline_version }}"
     - foreman_installer
 
-- hosts: pipeline-{{ pipeline_type }}proxy-{{ pipeline_version }}-{{ pipeline_os }}
+- hosts: "{{ forklift_proxy_name }}"
   become: yes
+  vars_files:
+    - vars/install_base.yml
+    - vars/{{ pipeline_type }}_base.yml
```

Comment:
this overrides the scenario selection.

Reply:
yepp, re-running the pipe with the two type lines removed and it passes

Reply:
Removed now but perhaps we shouldn't set variables directly but rather have a `server_box_installer_parameters`. This would also allow us to use the same pipeline for both Foreman and Katello. Leaving that for a future iteration.